content/en/docs/usage/make_build.md

```diff
@@ -461,3 +461,455 @@ oOo oOO| | | | |   (| | | (_) |  _) :_
+```
+
+1. Choose the `x86` architecture from `Architecture Selection`
+
+1. Choose the `KVM` platform from `Platform Configuration`
+
+1. Choose all `Virtio` options from `Platform configuration->KVM guest->Virtio`
+
+1. Choose the `SQLite`, `musl` and `vfscore` libraries from the `Library Configuration` screen.
+
+1. Under the `vfscore` library, choose the `Default root filesystem` to be `9pfs`
+
+We'll build our app:
+
+```console
+$ make
+```
+
+We shuold now create a directory named `fs0` in the `app-sqlite` directory.
+Inside it we'll create this `script.sql` script:
+
+```sql
+CREATE TABLE tab (d1 int, d2 text);
+INSERT INTO tab VALUES (random(), cast(random() as text)),
+(random(), cast(random() as text)),
+(random(), cast(random() as text)),
+(random(), cast(random() as text)),
+(random(), cast(random() as text)),
+(random(), cast(random() as text)),
+(random(), cast(random() as text)),
+(random(), cast(random() as text)),
+(random(), cast(random() as text)),
+(random(), cast(random() as text));    
+```
+
+The last step is running our app:
+
+```console
+$ sudo qemu-system-x86_64 -fsdev local,id=myid,path=$(pwd)/fs0,security_model=none \
+                          -device virtio-9p-pci,fsdev=myid,mount_tag=rootfs,disable-modern=on,disable-legacy=off \
+                          -kernel "build/app-sqlite_kvm-x86_64" \
+                          -enable-kvm \
+                          -nographic
+```
+
+And testing it:
+
+```console
+Booting from ROM..Powered by
+o.   .o       _ _               __ _
+Oo   Oo  ___ (_) | __ __  __ _ ' _) :_
+oO   oO ' _ `| | |/ /  _)' _` | |_|  _)
+oOo oOO| | | | |   (| | | (_) |  _) :_
+ OoOoO ._, ._:_:_,\_._,  .__,_:_, \___)
+      Epimetheus 0.12.0~4c7352c0-custom
+SQLite version 3.30.1 2019-10-10 20:19:45
+Enter ".help" for usage hints.
+Connected to a transient in-memory database.
+Use ".open FILENAME" to reopen on a persistent database.
+sqlite> .read script.sql
+sqlite> select * from tab;
+-7215681294144457603|-7881854117727259003
+2117848104624103428|-1622087965530640476
+-3409481001054928427|-6710007935272102091
+-8864771657867452020|4690809563575391968
+799097535707348776|-9015585007893225196
+-6011599994684870502|5715089397278453373
+-861992803967016915|1583603794332302036
+-5556545375765846451|-8514616268392191775
+6518576960059048853|6826969221610647599
+-2505304097900087778|-1087908404908377458
+sqlite> .exit
+```
+
+**Pro tip**: We usually save the launch command in a script.
+This way, it's easier to type it, but if you fancy even more automation, check out [`qemu-guest`](https://github.com/unikraft/kraft/blob/staging/scripts/qemu-guest) or even [kraft](https://unikraft.org/docs/usage/init/#kraft-run).
+
+### `app-sqlite` for `AArch64`
+
+The process of building `app-sqlite` for `AArch64` is quite similar to the one we've layed out for the `x86` architecture, the only thing that needs changes being the architecture, which should be switched to `Armv8` in the `Architecture Selection` menu.
```

Comment:
Maybe also specify that a properclean must be done after changing the architecture.